### PR TITLE
refactor: Rebuild ArtworkBanner in v3

### PR DESCRIPTION
**Pull request by damassi**
This refactors the ArtworkBanner component and applies the v3 styles: 

Fair: 

Screen Shot 2021-07-28 at 12 42 55 PM

Auction: 

Screen Shot 2021-07-28 at 12 43 03 PM

Show: 

Screen Shot 2021-07-28 at 12 51 35 PM

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Please use `<SystemQueryRenderer />` instead of `<QueryRender />`. This prevents double fetching during the server-side render pass. See:
> `src/v2/Apps/Artwork/Components/ArtworkBanner/ArtworkBanner.tsx`
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against bf38528e9b588764c2e0563ea7207590c61983bb
</p>

File: src/v2/Apps/Artwork/Components/ArtworkBanner/index.tsx [hunk 1]
```diff
@@ -1,174 +0,0 @@
...
-}
-
-export const ArtworkBanner: React.SFC<ArtworkBannerProps> = props => {
-  const { context, partner, sale } = props.artwork
-
-  if (!context) {
-    return null
-  }
-
-  switch (context.__typename) {
```
Comment (damassi) [inline src/v2/Apps/Artwork/Components/ArtworkBanner/index.tsx:22]: mostly just cleaned up this stuff and renamed file

File: src/v2/Apps/Artwork/Components/ArtworkBanner/ArtworkBanner.tsx [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+
+  if (!bannerProps) {
+    return null
+  }
+
+  const { image, meta, name, subHeadline, href } = bannerProps
+  const isShow = props.artwork?.context?.__typename === "Show"
+
+  return (
+    <RouterLink to={href} textDecoration="none">
```
Comment (dzucconi) [inline src/v2/Apps/Artwork/Components/ArtworkBanner/ArtworkBanner.tsx:35]: I'm not sure if it matters but I think this anchor tag is collapsed without `display: block;`?

**Review by dzucconi — APPROVED** [review]

File: src/v2/Apps/Artwork/Components/ArtworkBanner/ArtworkBanner.tsx (hunk 2 above)
Comment (damassi) [reply src/v2/Apps/Artwork/Components/ArtworkBanner/ArtworkBanner.tsx:35]: Nah its all good

**Pull request merged**